tests/scenarios/v2-addon-dev-watch-test.ts

```diff
@@ -51,14 +51,19 @@ Scenarios.fromProject(() => baseV2Addon())
 
           plugins: [
             addon.publicEntrypoints(['components/**/*.js']),
+
             addon.appReexports(['components/**/*.js']),
-            addon.gjs(),
-            addon.hbs(),
+
             addon.dependencies(),
-            addon.publicAssets('custom-public'),
 
             babel({ babelHelpers: 'bundled' }),
 
+            addon.hbs(),
```

Comment:
I think babel must be the last. whats the issue with the order?

Reply:
ah, I see, thats how the blueprint is.
But the recent PR chanhed hbs & gjs plugins to use transform hook. So the order of the blueprint must be changed.
the tests in embroider were already at correct order before the change... I wonder why it wasnt the same order in the blueprint

Reply:
looks like the PR i did was more of a breaking change then?

Reply:
well, the blueprint should really have the babel plugin as last.

Reply:
@patricklx ready to go now, see the PR description! 

> well, the blueprint should really have the babel plugin as last.

We can still decide to make that change in the blueprint if we feel this is the safer path forward. But the change in this PR makes this optional, which I think is better than forcing this on users by running into build errors...